src/backend/auth.api.ts

```diff
@@ -117,6 +117,50 @@ const loginUser = async (userData: any) => {
   }
 };
 
+
+/**
+ * @description send a link to email for updating user's password
+ * @param {string} userData
+ * @returns {Object} response
+ */
+const forgotpassword = async (email: string) => {
+  try {
+    // console.log("login:", userData?.email, userData?.password);
+    if (!email) {
+      throw new Error("email is needed ");
+    }
+    const response = await account.createRecovery(email, `http://localhost:3000/updatepassword`);
```

Comment:
Please don't hard code the URL

Reply:
I too don't want it to however whenever I use baseURL from or proccess.env.NEXT_PUBLIC_BASE_URL, I am getting this error
`Invalid url param: URL host must be one of: localhost, cloud.appwrite.io`